examples/query-playground/src/components/Editor.js

```diff
@@ -1,5 +1,4 @@
 import React from 'react'
-
 import AceEditor from 'react-ace'
 
 import 'brace/mode/json'
```

Comment:
Not in the scope of this PR, I'm just curious: Do we want import statements without identifiers below the other imports and separated with a space? Or is this accidentally enforced this way by the import sort eslint rules plugin?

Reply:
Not sure... looks like this was just an extra newline that wasn't removed by the rule?